moped-editor/src/views/projects/projectView/ProjectPhase/helpers.js

```diff
@@ -177,3 +184,21 @@ export const onSubmitPhase = ({
     refetchQueries: ["ProjectSummary"],
   }).then(() => onSubmitCallback());
 };
+
+export const onSubmitStatusUpdate = ({ data, mutate }) => {
+  const { status_update, project_id, phase_id, user_id } = data;
+
+  mutate({
+    variables: {
+      objects: [
+        {
+          project_note: DOMPurify.sanitize(status_update),
+          project_id,
+          added_by_user_id: user_id,
+          project_note_type: STATUS_UPDATE_TYPE_ID,
+          phase_id,
```

Comment:
@johnclary iirc the phase added to a status update is the current phase. What are your thoughts on how this should behave? I think that I missed the mark on the business logic by passing the phase selected in form on submission.

Reply:
ah, right. we do indeed want the status update tagged with the project's current phase, so the behavior here would mirror the behavior from status updates elsewhere in the app.

Reply:
Cool - I pushed an update to store the current phase of the project in the new note record unless the user is submitting a new current phase.